english/youre-fine/youre-fine.md

```diff
@@ -1,3 +1,4 @@
 Whew, you're fine. It was just chocolate.
 
 [Back to the room...](../marshmallow.md)
+[Relax with a haiku...](../hiku.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.